swarm/network/protocol_test.go

```diff
@@ -33,6 +33,8 @@ import (
 const (
 	TestProtocolVersion   = 8
 	TestProtocolNetworkID = 3
+
+	noLightNode = false
```

Comment:
Why is it called `noLightNode` now? It is a negation, so if set to `false` it means `true`, so it means it is a light node!
At least that is what I  understand and I would prefer a name with no negation

Reply:
The rationale was that this was a `const`ant  and was specifying the intention of the value. That way when someone sees `newBzzHandshakeTester(1, addr, noLightNode)` they know that the `lightNode` parameter is set to false.

The alternatives are
 ```
newBzzHandshakeTester(1, addr, false)
```
where it is not clear what is being set to false. Or

 ```
const lightNode = false

// lots of code

newBzzHandshakeTester(1, addr, lightNode)
```
Where someone has to go back to the definition of `lightNode` to see it's false. Of course the value can be toggled, which would then affect all of the methods that use that constant (and may not have intentionally needed the value change.

All of this is to say: How about `lightNodeDisabled`?

Reply:
My position is that the function has this signature:
` func newBzzHandshakeTester(n int, addr *BzzAddr, lightNode bool)`

which means it can be called with different configurations - `lightNode == false` or `lightNode == true`, so actually creating a `const` isn't really conclusive, if we do that then there should be a second `const` for `lightNodeEnabled`, or even better, an `enum` equivalent.

But if you insist in one single `const` then `lightNodeDisabled` is better for me.